tee-worker/core-primitives/stf-interface/src/lib.rs

```diff
@@ -20,6 +20,12 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
+#[cfg(all(feature = "std", feature = "sgx"))]
+compile_error!("feature \"std\" and feature \"sgx\" cannot be enabled at the same time");
+
+#[cfg(all(not(feature = "std"), feature = "sgx"))]
+extern crate sgx_tstd as std;
```

Comment:
Is this really needed?

Reply:
It _was_ when I was passing the version numbers as `String` (as opposed to `u32`) :D
Will try yeeting it; thanks for the catch :raised_hands:

Reply:
Yeeted :D